scripts/bootstrap-master-4.pp

```diff
@@ -53,8 +53,8 @@
   manage_modulepath => false,
   sources           => {
     'gitlab' => {
-      'remote'  => 'https://github.com/genebean/control-repo.git',
-      'basedir' => "/etc/puppetlabs/code/environments",
+      'remote'  => 'https://github.com/thespain/control-repo.git',
```

Comment:
Yes, and the intention is to change this back to your control repo, however just for the purposes of testing the fully automated setup I changed this temporarily.